apps/src/applab/applab.js

```diff
@@ -983,6 +984,10 @@ Applab.reset = function (first) {
  */
 function runButtonClickWrapper(callback) {
   $(window).trigger('run_button_pressed');
+
+  // Change back to default screen before we serialize. This also makes it so
+  // that we'll be back on the default screen after a reset.
+  studioApp.reduxStore.dispatch(changeScreen(elementUtils.getDefaultScreenId()));
```

Comment:
Doesn't this also fire when you click 'Run'?  I think we already have a behavior somewhere that sends you to the default screen on run, and while I'm sure doing so twice isn't very harmful right now, I wonder next time someone wants to change that behavior if they'll find both places.